python/health-check/src/health_check/config.toml

```
@@ -0,0 +1,19 @@
+[podman]
+network_name = "health-check-network"
+
+[promtail]
+container_name = "health_check_promtail"
+image = "registry.opensuse.org/systemsmanagement/uyuni/master/containers/uyuni/health-check-promtail:latest"
+
+[loki]
+container_name = "health_check_loki"
+jobs = ["cobbler", "postgresql", "rhn", "apache"]
+image = "registry.opensuse.org/systemsmanagement/uyuni/master/containers/uyuni/health-check-loki:latest"
+
+[exporter]
+container_name = "health_check_supportconfig_exporter"
+image = "registry.opensuse.org/systemsmanagement/uyuni/master/containers/uyuni/health-check-supportconfig-exporter:latest"
```

When we release it to stable, the container will be available from `image = "registry.opensuse.org/uyuni/`
```suggestion
image = "registry.opensuse.org/uyuni/health-check-supportconfig-exporter:latest"
```